tests/test_include_aws.cpp

```diff
@@ -15,6 +15,30 @@ void test_include_aws(void **unused) {
 }
 
 int main(void) {
+
+#ifdef __APPLE__
```

Comment:
why add this in this PR?

Reply:
@sfc-gh-azhan 
The master has this changes for a put/get issue. And his branch was cut before that got merged. 
Ideally jenkins merges this PR with master and runs tests. But somehow this PR checks still see put/get test failing. 
So recommended him to merge with master to see if that fixes the Checks failing.